torch/quantization/fx/quantize.py

```diff
@@ -781,6 +791,9 @@ def load_arg(a):
         quantized_root = quantized
         quantized_graph = quantized.graph
         for node in quantized_graph.nodes:
+            if node.op == 'output':
```

Comment:
This is probably not needed because there is a `node_copy` in the else branch which will do the same thing.